spacemesh/v1/admin_types.proto

```diff
@@ -131,6 +135,29 @@ message EventMalfeasance {
 
 message EventStreamRequest {}
 
+message EventPoetClientCreated {
+  string url = 1;
+}
```

Comment:
I know this is part of the issue but this seems very technical and has little meaning to a user/operator: It only says that the node created an object internally with which it could connect to a PoET if it needs to and only happens during startup of the node (I think).

Maybe instead of removing we could also rename this event to something less technical: "`url` added to list of PoETs" or something similar.

Wdyt?

Reply:
The argument for not having this event is: if the config entry with the URL would be invalid or at the wrong place of the config, the node would refuse to start.

So if as a user/operator I add a new poet to the config and the node starts successfully the client for it was created.

Reply:
With my current changes the whole sequence of evens looks like:
```
Event_PoetClientCreated
Event_Beacon
Event_InitStart
Event_InitComplete
Event_PostServiceStarted
Event_PostStart
Event_PostComplete
Event_PoetWaitRound
Event_RegisteredInPoet
Event_PoetWaitProof
Event_PoetProofReceived
Event_BestPoetProofSelected
Event_PostStart
Event_PostComplete
Event_AtxPublished
```
(taken from node/node_test.go). At the moment all that looks "quite technical" :)

dunno if info about client creation is useful. More important is info about registration.

I can adapt that to any form you will suggest, but i need a bit more precise an input from someone who will be using that. 
It's right moment to tune all event names. E.g. maybe instead of `PoetProofReceived` there should `ProofDownloadedFromPoet`?

Technically event about poet client creation is emitted right after successful client creation. If it has to be renamed in proposed way, imo that will also require moving it in the node code to the place where poet service is added to poetClients list on node/node.go, in `initServices` method?

Reply:
`Event_Beacon` is arguably also a technical detail that doesn't need to be reported, but the others make sense to tell a user about:

- `InitStart`: Post initialization started, the node now creates post data for the identity it uses
- `InitComplete`: Post initialization completed, the node is now ready to participate in consensus
- `PostServiceStarted`: can be argued if needed or not, post is now handled by a dedicated process that can be run "supervised" (i.e. the node starts and manages that process) or manually by the user. If a user wants to manage multiple identities they have to start one post service per identity. I believe this event is still useful in communicating to the user that the node manages an identity and started post (service) for it.
- `PostStart`: The node has started generating a post proof (in your test most likely the initial post for the identity it manages)
- `PostComplete`: The node has completed generating a post proof

All of these can happen multiple times (once per configured PoET):
- `PoetWaitRound`: The node now wants to register at a PoET and waits until a registration window opens
- `PoetWaitProof`: The node registered at a PoET and is now waiting to receive a proof from it
- `PoetProofReceived`: The node successfully downloaded a proof from a PoET

Then the node does another post and publishes it's first ATX, after that events will be poet events, post events, atx publish, poet events, post events, atx publish, aso.

Now describing it: `RegisteredInPoet ` and `PoetWaitProof` are kinda saying the same thing,

Reply:
I believe `Event_PoetClientCreated` can be dropped completely unless @poszu objects.

Reply:
Sure, it's not necessary. AFAIR, the idea was to give the user a confirmation about the poets that the given node will use.